### PR TITLE
db410c: aosp guide update with recent build and flash instructions

### PR DESCRIPTION
**Pull request by ric96**
Signed-off-by: Sahaj Sarup <sahaj.sarup@linaro.org>

### CODE REVIEW

File: consumer/dragonboard410c/guides/aosp.md [hunk 1]
```diff
@@ -15,69 +15,31 @@ The following instructions can be used to attempt a build based on the AOSP mast
...
 Then run:
 ```shell
-repo init -u https://android.googlesource.com/platform/manifest -b master
-cd .repo
-git clone https://github.com/robherring/android_manifest.git -b master local_manifests
-cd ..
-repo sync -j24
+$ repo init -u https://android.googlesource.com/platform/manifest -b master
+$ git clone https://android-git.linaro.org/git/platform/manifest.git .repo/local_manifests -b dragonboard
+$ repo sync -jX -c # X being the number of logical cores in you build system
```
Comment (ndechesne) [inline consumer/dragonboard410c/guides/aosp.md]: i like to recommend -j`nproc` in such situations.

File: consumer/dragonboard410c/guides/aosp.md [hunk 2]
```diff
@@ -15,69 +15,31 @@ The following instructions can be used to attempt a build based on the AOSP mast
...
-cd ..
-git clone https://git.kernel.org/pub/scm/linux/kernel/git/robh/linux.git -b qcom-4.14 <KERNEL_TOP>
-cd <KERNEL_TOP>
-make ARCH=arm64 db410c_android_defconfig android-base.config android-recommended.config
-make CROSS_COMPILE=aarch64-android-gnu- ARCH=arm64 all
-cat arch/arm64/boot/Image.gz arch/arm64/boot/dts/qcom/apq8016-sbc.dtb > Image.gz-dtb
-mkbootimg --kernel Image.gz-dtb --ramdisk <ANDROID_TOP>/out/target/product/linaro_arm64/ramdisk.img --output boot-db410c.img --pagesize 2048 --base 0x80000000 --cmdline 'androidboot.selinux=permissive firmware_class.path=/system/vendor/firmware/'
+$ source build/envsetup.sh
+$ lunch db410c32_only-userdebug
+$ make -jX # X being the number of logical cores in you build system
```
Comment (ndechesne) [inline consumer/dragonboard410c/guides/aosp.md]: ditto

File: consumer/dragonboard410c/guides/aosp.md [hunk 3]
```diff
@@ -15,69 +15,31 @@ The following instructions can be used to attempt a build based on the AOSP mast
...
-Note that using this script is a temporary solution until the proper firmware package is published on 96boards website.
-
 # Building AOSP
 ```shell
-source build/envsetup.sh
-lunch linaro_arm64-userdebug
-make -j24
-```
-
-# Building Kernel
```
Comment (ndechesne) [inline consumer/dragonboard410c/guides/aosp.md]: i think we need to keep a section about kernel. while the default build will use pre-built, we want to tell users how to recompile their own kernel. is that something you can add?

**Comment by Mani-Sadhasivam** [comment]
+1 on using -jnproc. There are still some places in the documentation where we specify hardcoded threads.

File: consumer/dragonboard410c/guides/aosp.md (hunk 1 above)
Comment (ric96) [reply consumer/dragonboard410c/guides/aosp.md]: ok i'll add -j$(nproc)

File: consumer/dragonboard410c/guides/aosp.md (hunk 3 above)
Comment (ric96) [reply consumer/dragonboard410c/guides/aosp.md]: can you send/link me the instructions for building aosp kernel?

Comment (ndechesne) [reply consumer/dragonboard410c/guides/aosp.md]: can you please check with Amit P.?

**Comment by ric96** [comment]
@ndechesne any updates on this

**Comment by ndechesne** [comment]
looks good to me. i don't find a way I can 'approve' it..

**Comment by ric96** [comment]
@sdrobertw ping

**Pull request merged**